portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,130 @@
 <!DOCTYPE html>
 <html>
   <head>
+
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+
+    <title>Shivani Ghanta</title>
+
+    <!-- CSS -->
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="bootstrap.css">
+
+    <!-- Fonts -->
+    <link href='https://fonts.googleapis.com/css?family=Aclonica:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Adamina:400,100,200,300,500,600,700,800,900' rel='stylesheet' type='text/css'>
+
     <script src="script.js"></script>
+
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+      <!-- Navigation Bar -->
+      <nav class="navbar navbar-default">
+          <div class="container">
+              <div class="navbar-header">
+                  <div data-toggle="collapse" data-target="#myNavbar">
+                      <span class="sr-only">Toggle navigation</span>
+                      <span class="icon-bar"></span>
+                      <span class="icon-bar"></span>
+                      <span class="icon-bar"></span> 
+                  </div>
+              </div>
+
+    <div id="navbar" class="navbar-collapse">
+        <ul class="nav navbar-nav">
+            <li><a href="index.html">Portfolio</a></li>
+            <li><a href="mailto:shivani.ghanta@gmail.com">Email Me</a></li>
+            <li><a href="resume.pdf">Resume</a></li>
+        </ul>
     </div>
+
+</div>
+      </nav>
+
+      <!-- Title Content & Bio -->
+      <div class="bitmoji row">
+              <div class="container">
+                  <div class="row">
+                      <div class="about">
+                          <div id="photos">
+                              <center><img id = "bitmoji" src="bitmoji-icon.png" class="img-responsive" height="420" width="420"/></center>
```

Comment:
I noticed in the screenshot that the images do not show - could you also include the image files in this PR and update the screenshot?